src/symengine/blocks.jl

```diff
@@ -137,3 +136,7 @@ function SymEngine.subs(::Type{T}, c::AbstractBlock, args...; kwargs...) where {
     c = chiparams(c, map(x -> T(subs(x, args...; kwargs...)), getiparams(c))...)
     chsubblocks(c, [subs(T, blk, args..., kwargs...) for blk in subblocks(c)])
 end
+
+# dumpload
+YaoBlocks.tokenize_param(param::Basic) = QuoteNode(Symbol(param))
```

Comment:
why dump parameters without handling its scope? this will conflict as we discussed before.

Reply:
We don't have SymEngine dependancy in YaoBlocks.